Changelog.md

```diff
@@ -4,20 +4,32 @@
 
 - Support for big-endian architectures.
 - Added `unicodeVersion`.
-- Added the module `Unicode.Char.Case.Compat`.
 - Added `GeneralCategory` data type and corresponding `generalCategoryAbbr`,
   `generalCategory` functions.
 - Added the following functions to `Unicode.Char.General`:
   `isAlphabetic`, `isAlphaNum`,
   `isControl`, `isMark`, `isPrint`, `isPunctuation`, `isSeparator`,
   `isSymbol` and `isWhiteSpace`.
 - Added the module `Unicode.Char.Numeric`.
-- **Breaking change:** Changed the behavior of `isLetter` and `isSpace` to match
-  `base`’s `Data.Char` behavior. Move these functions to the compatibility module
-  `Unicode.Char.General.Compat`. The previous behavior is obtained using
-  `isAlphabetic` and `isWhiteSpace` respectively.
+- Add compatibility modules:
+
+  - `Unicode.Char.General.Compat`
+  - `Unicode.Char.Case.Compat`
+
+  These modules respect the behavior of `base:Data.Char`.
 - Re-export some functions from `Data.Char` in order to make `Unicode.Char`
   a drop-in replacement.
+- Drop support for GHC 7.10.3
+
+### Deprecations
+
+- In `Unicode.Char.General`.
+
+  - `isLetter`
+  - `isSpace`
+
+  Preserve the behavior of these functions in `isAlphabetic` and `isWhiteSpace`
+  respectively.
```

Comment:
We still have a breaking change in `Unicode.Char`. The behavior of isLetter and isSpace that are exported through Unicode.Char has changed. It should be mentioned as a breaking changed.

Also, given the above I don't know how much useful the deprecation is. Deprecation makes the behavior different when these are imported from Unicode.Char vs Unicode.Char.General which may not be desirable.

Reply:
Maybe we could make a release without exposing the new `base`-compatible functions in `Unicode.Char`, then:
- bump the versión to 0.4.0,
- remove the deprecated functions and
- expose the`base`-compatible functions.

Reply:
yeah, sounds good to me. We can expose these in the next major release.

Reply:
Ok. I updated #57